modules/oadp-ceph-cephfs-back-up-dba.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-ceph-cephfs-back-up-dba_{context}"]
+= Creating a DPA for use with CephFS storage
+
+You need to create a Data Protection Application (DPA) CR before you use the OpenShift API for Data Protection (OADP) 1.2 Data Mover to back up and restore data using CephFS storage.
+
+.Procedure
+
+. Verify that `deletionPolicy` of the `VolumeSnapshotClass` CR is set to `Retain` by running the following command:
++
+[source,terminal]
+----
+$ oc get volumesnapshotclass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"Retention Policy: "}{.deletionPolicy}{"\n"}{end}'
+----
+
+. Verify that the labels of the `VolumeSnapshotClass` CR are set to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc get volumesnapshotclass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"labels: "}{.metadata.labels}{"\n"}{end}'
+----
+
+. Verify that the `storageclass.kubernetes.io/is-default-class` annotation of the `StorageClass` CR is set to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc get storageClass -A  -o jsonpath='{range .items[*]}{"Name: "}{.metadata.name}{"  "}{"annotations: "}{.metadata.annotations}{"\n"}{end}'
+----
+
+. Create a Data Protection Application (DPA) CR similar to the following.
++
+.Example DPA CR
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
```

Comment:
```suggestion
          bucket: <my_bucket>
```
See the[ repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) on variables (you need to scroll a little).

Reply:
Thanks!